discovery-provider/src/app.py

```diff
@@ -415,7 +415,7 @@ def configure_celery(flask_app, celery, test_config=None):
             },
             "index_trending": {
                 "task": "index_trending",
-                "schedule": crontab(minute=15, hour="*"),
+                "schedule": timedelta(seconds=60),
```

Comment:
this should be cheap. can we switch to 10s so we do our best job at staying as on time as possible?